app/routes/list.js

```diff
@@ -22,7 +22,7 @@ export default class ListRoute extends Route.extend(DataTableRouteMixin) {
   async model(params) {
     const reglements = await this.store.query('regulatory-statement', {
       filter: { folder: RS_STANDARD_FOLDER },
-      include: ['document.currentVersion'],
+      include: ['published-version'],
```

Comment:
So here I think is my bad, but I think we need document.current-version, if not we for sure don't need the publish version on the list (or at least that's what I think please double check)

Reply:
I removed it for now, we can always add a delete option on the edit page later.

Reply:
Yes I think so too, but I think the correct syntax is `include: 'document.currentVersion'`